.circleci/config.yml

```diff
@@ -12,13 +12,15 @@ jobs:
 
   checkESLint:
     docker:
-      - image: node:slim
+      - image: node:18
```

Comment:
the slim containers sadly don't have yarn >2 installed, so needed to switch this. Some packages are local dependencies of each other, so they will need to be built with yarn workspaces, and then linted.